libraries/Microsoft.Bot.Schema/Teams/TeamsChannelData.cs

```diff
@@ -19,6 +19,25 @@ public TeamsChannelData()
             CustomInit();
         }
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="TeamsChannelData"/> class.
+        /// </summary>
+        /// <param name="channel">Information about the channel in which the message was sent.</param>
+        /// <param name="eventType">Type of event.</param>
+        /// <param name="team">Information about the team in which the message was sent.</param>
+        /// <param name="notification">Notification settings for the message.</param>
+        /// <param name="tenant">Information about the tenant in which the
+        /// message was sent.</param>
+        public TeamsChannelData(ChannelInfo channel = default, string eventType = default, TeamInfo team = default, NotificationInfo notification = default, TenantInfo tenant = default)
+        {
+            Channel = channel;
+            EventType = eventType;
+            Team = team;
+            Notification = notification;
+            Tenant = tenant;
+            CustomInit();
+        }
+
```

Comment:
To avoid repeating the code, we can make the old constructor call the new overload with `: this(channel, eventType, team, notification, tenant, null)`. In the new constructor you'll have the properties setting. (Same we discussed in today's SUP for my SuggestedActions problem).

Reply:
solved